content/en/tracing/setup_overview/setup/dotnet-core.md

```diff
@@ -285,13 +263,61 @@ When using `systemctl` to run .NET applications as a service, you can also set e
 
 [1]: /agent/guide/agent-configuration-files/#agent-main-configuration-file
 [2]: https://www.freedesktop.org/software/systemd/man/systemctl.html#set-environment%20VARIABLE=VALUE%E2%80%A6
+
+### Configure the Datadog Agent for APM
```

Comment:
This section appears under "Automatic instrumentation," but it applies to all instrumentation. Note that this issue was already present before this PR.